talon/signature/constants.py

```diff
@@ -1,2 +1,2 @@
 SIGNATURE_MAX_LINES = 11
-TOO_LONG_SIGNATURE_LINE = 60
+TOO_LONG_SIGNATURE_LINE = 100
```

Comment:
Are we seeing signatures over 60 characters?  That's impressive!

Reply:
Yeah, so some of them are long when converted to text. I think I recall that Haggai's was one of them.